p2p/peer.go

```diff
@@ -246,6 +256,19 @@ func (p *peer) String() string {
 //---------------------------------------------------
 // Implements service.Service
 
+func (p *peer) Metrics() *Metrics {
+	return p.metrics
+}
+
+func (p *peer) ChIDToMetricLabel(chID byte) string {
+	//return p.mlc.ChIDToMetricLabel(chID)
+	return "" // TODO: port also this change
+}
```

Comment:
this could be deleted @evan-forbes right?

Reply:
I don't remember tbh

probably as it looks like some comet compatability thing